src/Compilers/CSharp/Portable/Emitter/NoPia/EmbeddedTypesManager.cs

```diff
@@ -100,9 +112,16 @@ private MethodSymbol LazyGetWellKnownTypeMethod(ref MethodSymbol lazyMethod, Wel
             return lazyMethod;
         }
 
-        internal override int GetTargetAttributeSignatureIndex(Symbol underlyingSymbol, CSharpAttributeData attrData, AttributeDescription description)
+        internal override int GetTargetAttributeSignatureIndex(
+#if DEBUG
+            SymbolAdapter
```

Comment:
>SymbolAdapter [](start = 12, length = 13)

Could we do a conditional `using static SymbolAdapter = Symbol;` at the top of the file in release mode, to remove this conditional logic? Same for the other parameter ifdefs in this file?